packages/terra-dialog/src/Dialog.scss

```diff
@@ -3,4 +3,19 @@
     height: 100%;
     width: 100%;
   }
+
+  .header { // May not be needed if action header is already themeable.
+    border-bottom: var(--terra-dialog-header-bottom-border, none);
+  }
+
+  .dialog-body {
+    background-color: var(--terra-dialog-body-background-color, transparent);
```

Comment:
How about the foreground? Typically, when you set a background, you also need to set a foreground. Or will that be handled by another component?

Reply:
https://github.com/cerner/terra-core/pull/1209/commits/2fabef8d09a87699422ef4b44f00ce87220ae782